standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml

```diff
@@ -5,7 +5,7 @@
   </assembly>
   <members>
     <member name="T:System.Buffers.ArrayPool`1">
-      <summary>Provides a resource pool that enables reusing instances of type <see cref="T[]" />.</summary>
```

Comment:
Are this changes correct? Should we remove the references to `T[]` and/or `T`?

Reply:
All these files were built by the Docs repo. Let me find the xml that generated it.

Reply:
Yes, that's exactly how we merged it and how it was generated.
https://github.com/dotnet/dotnet-api-docs/blob/6f84948d2563a9b53a9cfb292d78a2caecacc2dd/xml/System.Buffers/ArrayPool%601.xml#L1

We could improve it, but the change will go into a future .NET Core IntelliSense version.

Reply:
The cref link was removed on purpose in this PR:
https://github.com/dotnet/dotnet-api-docs/pull/3466